join-statements/joinstatements.md

```diff
@@ -4,63 +4,33 @@
 
-Similar to the setup of the INNER JOIN, except if we do a LEFT OUTER JOIN instead, we would get the students already in the extracurricular society and also any student that is not in the extracurricular activity. This could be good in order to see which students could be good recruiting targets.
+**RIGHT OUTER JOIN** - similar to the LEFT INNER JOIN. Lets assume table A is specified in the `FROM` statement, and table B is specified in the `RIGHT OUTER JOIN` statement. In the query, all columns from table B will be represented in the output. Columns pertaining to table A will have values resulting from an `INNER JOIN` if the corresponding conditional statement specified in the `ON` section is TRUE and will be joined with the corresponding row from table B. If the conditional statment specified in the `ON` section is FALSE, columns pertaining to table A will show up as `NULL` values.
 
-```
-SELECT students.name societies.society
-FROM students
-LEFT JOIN societies ON students.id = societies.student_id;
-```
-
-**RIGHT OUTER JOIN** - similar to LEFT OUTER JOIN, except instead of focusing on the table specified in the `FROM` statement (in the previous case it was table A), we would focus on the table linked to the  `RIGHT OUTER JOIN` statement, in this case, table B. Similarly, there will be a single row in the output for every row in table B, and any row that does not meet the conditional in the `ON` statement will have null values in the output values of columns from table A.
-
-In the extracurricular society example shown before, using a RIGHT OUTER JOIN would yield an output similar to the INNER JOIN output, _plus_ if we've got societies with no members, they would show up. This would be useful for example to pinpoint societies for students interested in building a society from the ground up.
-
-```
-SELECT students.name societies.society
-FROM students
-RIGHT JOIN societies ON students.id = societies.student_id;
-```
+The head guidance counselor would like to get a list of all societies, with the students that are part of them, and just show the society name in order to reflect if the society has no students.
 
 **FULL OUTER JOIN** - It's the combination of an INNER JOIN, LEFT OUTER JOIN, and RIGHT OUTER JOIN.
 
-Similar to the LEFT OUTER JOIN, RIGHT OUTER JOIN, and the INNER JOIN. We would get the output of the LEFT JOIN, and the RIGHT JOIN and would be useful in being able to easily find the students that have no society, and perhaps also point out to them which societies they would be able to make the most impact in by helping create it from the ground up.
-
-```
-SELECT students.name societies.society
-FROM students
-FULL OUTER JOIN societies ON students.id = societies.student_id;
-```
+A guidance counselor wants a list of all students and all societies, and wants to see which students are linked to each society, which student is not linked to a society, as well as which society lacks any students.
```

Comment:
you're still not adding a reason here even though you added to the example

Reply:
done